src/declarations.d.tsx

```diff
@@ -0,0 +1,5 @@
+// This entire file will be deleted once we have typescript definitions
+// for @openmrs/esm-api published to npm
+declare type Observable<T> = import("rxjs").Observable<T>;
```

Comment:
Why is this needed? Doesn't the `rxjs` package already have definitions for observables?

Reply:
Copy-paste detritus